src/rollback_to_stable/rts_btree.c

```diff
@@ -1016,22 +1032,36 @@ __rts_btree_abort_row_leaf(WT_SESSION_IMPL *session, WT_REF *ref, wt_timestamp_t
     WT_ROW_FOREACH (page, rip, i) {
         stable_update_found = false;
         if ((upd = WT_ROW_UPDATE(page, rip)) != NULL) {
+            __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_4,
+              WT_RTS_VERB_TAG_UPDATE_CHAIN_ABORT
+              "aborting any unstable updates on the update chain with rollback_timestamp=%s",
```

Comment:
We are unsure in the following code flow that we will abort the update. So instead of `abort`, how about using the `verify`? Also, try to print the `key` for verifying the update list.

Reply:
Sure thing! [2ff1584](https://github.com/wiredtiger/wiredtiger/pull/8960/commits/2ff15849792bdb0a1de4f07f3f8cb187db8e72a2)